docs/translations/api-docs/portal-unstyled/portal-unstyled-fr.json

```diff
@@ -0,0 +1,9 @@
+{
+  "componentDescription": "Portals provide a first-class way to render children into a DOM node\nthat exists outside the DOM hierarchy of the parent component.",
```

Comment:
These translation files seem to be orphan now. If there are not, we have a bug in the generation of the API 🙃

Reply:
Ye, it's a bug even deleting them and generating docs again it's producing this file.

Reply:
Done